### ISSUE TITLE
Decision Network输出端与论文描述似乎不符

### ISSUE THREAD

**Issue by sy8008** [Wslsdx/Deep-Learning-Approach-for-Surface-Defect-Detection#25]
你好！
在阅读model.py 时发现，代码中decision network的输出是对66x1维的特征计算softmax值（相当于作为一个66类的分类问题处理），最后输出实际上是概率最大的类别的索引。但论文中的描述是对66x1维的向量进行加权进而得到一个概率值，对应的应该是1个沿着channel进行的1x1卷积操作。
 vector=tf.concat([vector1,vector2,vector3,vector4],axis=3)
                    vector=tf.squeeze(vector,axis=[1,2])
                    logits = slim.fully_connected(vector, num_classes,activation_fn=None)
                    output=tf.argmax(logits,axis=1)

**Comment by DCC-lzhy** [comment]
对66x1维的向量进行加权进而得到一个概率值，这不就是softmax吗，没毛病啊，不是你后面说的对呀1x1卷积

**Comment by DCC-lzhy** [comment]
它最终输出的是一个概率值，就是【0,1】，判断是否是瑕疵，不是个多分类
